guides/git-usage.md

```diff
@@ -0,0 +1,72 @@
+---
+category: Tools
+expires: 2018-05-31
+title: Git Usage
+---
+
+## Overview
+[git][git] is a [distributed version-control system][dvcs]. It's distributed, in that there's no central source of truth. Each copy of a repository can contain the entire history, and be considered an authoritative source. But we tend to use it in a centralised fashion, treating the version that is [hosted on GitHub](https://github.com/ministryofjustice) as the shared central version, and basing all work against that copy.
+
+It has excellent tools for branching and merging (much better than Subversion). Branching is possible in Subversion, but merging can be very painful, so people used to be tentative about using that workflow. In git, that problem goes away.
```

Comment:
These lines are very long; the rest of the file appears to be broken at 100-120 columns. We're mostly doing 80 columns in this repo; while I'm not suggesting you stick to that, it might be good to be consistent within individual files.